packages/airnode-deployer/test/fixtures/config.aws.valid.json

```diff
@@ -54,7 +54,7 @@
     },
     "logFormat": "plain",
     "logLevel": "INFO",
-    "nodeVersion": "0.9.0",
+    "nodeVersion": "0.8.0",
```

Comment:
If this remains named as .valid.json it will get updated on next release and break again.

Reply:
It won't because the script is looking specifically for `config.valid.json` and this is `config.aws.valid.json`. I run the script and it was fine.

Reply:
Oh, okey.